golangci-lint/main.go

```diff
@@ -1,16 +1,13 @@
 package main
 
 import (
-	"fmt"
-	"os"
 	"strconv"
 )
 
 func main() {
-	i, err := strconv.Atoi("255")
-	if err != nil {
-		os.Exit(1)
-	}
+	strconv.Atoi("255")
 
-	fmt.Println(i)
+	strconv.Atoi("255")
```

Comment:
🚫 **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Error return value of `strconv.Atoi` is not checked (errcheck)